src/src/app/modules/identity/components/identity-login/identity-login.component.html

```diff
@@ -5,7 +5,8 @@ <h2>Login</h2>
   </div>
   <div class="form-group">
     <input type="password" class="form-control my-3" #Password="ngModel" name="Password" placeholder="Password" [(ngModel)]="formModel.password" required>
+    <p class="m-0">Forgor your password? <a href="" routerLink="/identity/forgot_password">Try to change your password</a></p>
```

Comment:
Bitte identifier in Form von einem Name Attribut oder Id Attribut hinzufügen. Bitte auch nochmal alle HTML files checken ob die Elemente ids haben. Dies vereinfacht die Testautomatisierung sehr

Reply:
done